styles/amplenote/catppuccin.user.css

```
@@ -0,0 +1,178 @@
+/* ==UserStyle==
+@name Amplenote Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/amplenote
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/amplenote
+@version 0.0.1
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/amplenote/catppuccin.user.css
+@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Aamplenote
+@description Soothing pastel theme for Amplenote
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire*", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+@-moz-document domain("amplenote.com") {
+    
+    :root:not(:has(body.theme-light)) {
+    #catppuccin(@darkFlavor, @accentColor);
+  }
+    :root:has(body.theme-light) {
+    #catppuccin(@lightFlavor, @accentColor);
+  }
+    
+
+    #catppuccin(@lookup, @accent) {
+        @rosewater: @catppuccin[@@lookup][@rosewater];
+        @flamingo: @catppuccin[@@lookup][@flamingo];
+        @pink: @catppuccin[@@lookup][@pink];
+        @mauve: @catppuccin[@@lookup][@mauve];
+        @red: @catppuccin[@@lookup][@red];
+        @maroon: @catppuccin[@@lookup][@maroon];
+        @peach: @catppuccin[@@lookup][@peach];
+        @yellow: @catppuccin[@@lookup][@yellow];
+        @green: @catppuccin[@@lookup][@green];
+        @teal: @catppuccin[@@lookup][@teal];
+        @sky: @catppuccin[@@lookup][@sky];
+        @sapphire: @catppuccin[@@lookup][@sapphire];
+        @blue: @catppuccin[@@lookup][@blue];
+        @lavender: @catppuccin[@@lookup][@lavender];
+        @text: @catppuccin[@@lookup][@text];
+        @subtext1: @catppuccin[@@lookup][@subtext1];
+        @subtext0: @catppuccin[@@lookup][@subtext0];
+        @overlay2: @catppuccin[@@lookup][@overlay2];
+        @overlay1: @catppuccin[@@lookup][@overlay1];
+        @overlay0: @catppuccin[@@lookup][@overlay0];
+        @surface2: @catppuccin[@@lookup][@surface2];
+        @surface1: @catppuccin[@@lookup][@surface1];
+        @surface0: @catppuccin[@@lookup][@surface0];
+        @base: @catppuccin[@@lookup][@base];
+        @mantle: @catppuccin[@@lookup][@mantle];
+        @crust: @catppuccin[@@lookup][@crust];
+        @accent-color: @catppuccin[@@lookup][@@accent];
+
+        color-scheme: if(@lookup =latte, light, dark);
```

```suggestion
 color-scheme: if(@lookup = latte, light, dark); 
  
 ::selection { 
   background-color: fade(@accent-color, 30%); 
 } 
  
 input, 
 textarea { 
   &::placeholder { 
     color: @subtext0 !important; 
   } 
 } 
```
Please add the remaining base styles:
https://github.com/catppuccin/userstyles/blob/3dcff2a7d7691c15a8efad16b23873d4654c885a/template/catppuccin.user.css#L78-L89